programs/st_client.c

```diff
@@ -78,7 +78,7 @@ get_milliseconds_count(void)
 	unsigned int milliseconds;
 
 	gettimeofday(&tv, NULL); /* get current time */
-	milliseconds = tv.tv_sec*1000LL + tv.tv_usec/1000; /* calculate milliseconds */
+	milliseconds = (unsigned int)(tv.tv_sec*1000LL + tv.tv_usec/1000); /* calculate milliseconds */
```

Comment:
Doesn't this always overflows nowadays? Avoiding these overflows by moving to 64-bit entities in https://github.com/sctplab/usrsctp/commit/7077e0ea9ad840d292a500552e7834db3164e3e7.